app/src/main/java/org/liberty/android/fantastischmemo/entity/Card.java

```diff
@@ -46,6 +46,9 @@
     @DatabaseField(defaultValue = "N/A", width = 8192)
     private String hint;
 
+    @DatabaseField(defaultValue = "false", width = 8192)
+    private Boolean favourite;
```

Comment:
why are you using the object Boolean instead of the primitive boolean? I think it would be preferred not to have the option of null.

Reply:
ok